.changeset/short-trainers-dance.md

```diff
@@ -0,0 +1,5 @@
+---
+"@nextui-org/theme": patch
+---
+
+Fix custom slot styling with twMerge true
```

Comment:
can you include the issue number here

`Fix custom slot styling with twMerge true (# 2153)`

Reply:
Good! It's done